app/models/imports/doc.rb

```diff
@@ -0,0 +1,6 @@
+module Imports
+  class Doc < Import
+    has_one :pdf, as: :parent, dependent: :destroy, autosave: true
```

Comment:
@mike-burns it is the autosave the reason it was not working last time?

Reply:
The autosave, and also the `belongs_to :parent` needed to be in the Import class.